src/connectors/range-slider/connectRangeSlider.js

```diff
@@ -34,9 +34,9 @@ Full documentation available at https://community.algolia.com/instantsearch.js/c
 
 /**
  * @typedef {Object} RangeSliderRenderingOptions
- * @property {function({min: number, max: number})} refine Sets a range to filter the results on. Both values
+ * @property {function(Array<number, number>)} refine Sets a range to filter the results on. Both values
  * are optional, and will default to the higher and lower bounds.
- * @property {{min: number, max: number}} numeric Results bounds without the current range filter.
+ * @property {{min: number, max: number}} range Results bounds without the current range filter.
```

Comment:
Can we "patch" this on our side to avoid breaking changes to the public API?

Reply:
It's not a breaking change, just a fix in the documentation.

- The `refine` function is already called with [an array and not an object](https://github.com/algolia/instantsearch.js/pull/2408/files#diff-6594f5647a134a790a0668875d7b1045L131).
- The property `numeric` doesn't exist on the render function, [it's name is `range`](https://github.com/algolia/instantsearch.js/pull/2408/files#diff-6594f5647a134a790a0668875d7b1045L210).

Reply:
True my bad, thought we were calling `refine({ min, max })`